### PR TITLE
Include polyfills based on browser list instead of usage

### PR DESCRIPTION
**Pull request by ljpengelen**
This addresses https://github.com/kabisa/maji/issues/251.

### CODE REVIEW

File: project_template/src/index.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { Provider } from "preact-redux";
 import FastClick from "fastclick";
 import "src/config/sentry";
 import store from "./store";
+import "@babel/polyfill";
```
Comment (pascalw) [inline project_template/src/index.js]: Maybe we could add a comment here pointing to the issue you created? So we instruct users that depending on your use cases you might want to remove this explicit polyfill import.

Comment (ljpengelen) [reply project_template/src/index.js]: I think it's cleaner to add this here (and rephrase the subtitle a little): https://github.com/kabisa/maji/blob/master/docs/README.md#components-you-might-not-need.

Comment (pascalw) [reply project_template/src/index.js]: Great! 👍

**Comment by pascalw** [comment]
Fixed conflict in Changelog. Merging.

**Pull request merged**